store/localcachelayer/channel_layer.go

```diff
@@ -167,3 +167,61 @@ func (s LocalCacheChannelStore) Get(id string, allowFromCache bool) (*model.Chan
 
 	return ch, err
 }
+
+func (s LocalCacheChannelStore) SaveMember(member *model.ChannelMember) (*model.ChannelMember, *model.AppError) {
+	member, err := s.ChannelStore.SaveMember(member)
+	if err != nil {
+		return member, err
+	}
+	s.InvalidateMemberCount(member.ChannelId)
+	return member, nil
+}
+
+func (s LocalCacheChannelStore) SaveMultipleMembers(members []*model.ChannelMember) ([]*model.ChannelMember, *model.AppError) {
+	members, err := s.ChannelStore.SaveMultipleMembers(members)
+	if err != nil {
+		return members, err
+	}
+	for _, member := range members {
+		s.InvalidateMemberCount(member.ChannelId)
+	}
+	return members, nil
+}
+
+func (s LocalCacheChannelStore) UpdateMember(member *model.ChannelMember) (*model.ChannelMember, *model.AppError) {
+	member, err := s.ChannelStore.UpdateMember(member)
+	if err != nil {
+		return member, err
```

Comment:
Nit: is it important to return `member` here? Caller already has the pointer.

Reply:
actually not, and if the `err != nil` `member` is going to be `nil` so I think is more explicit to put a `nil` there.